prow/github/client.go

```diff
@@ -253,3 +253,18 @@ func (c *Client) RemoveLabel(owner, repo string, number int, label string) error
 	}
 	return nil
 }
+
+func (c *Client) CloseIssue(owner, repo string, number int) error {
```

Comment:
You made the assumption that you "have everything you need from github" as an argument against go-github. It didn't hold true for more than half a day :-/